csrc/scheduler/multi_matmul.h

```diff
@@ -9,10 +9,64 @@
 
 #include <fusion.h>
 #include <scheduler/matmul_heuristic.h>
+#include <scheduler/mma_utils.h>
+#include <val_graph.h>
+#include <val_graph_visitor.h>
 #include <visibility.h>
 
 namespace nvfuser {
 
+// Base class for AmpereMultipleMatmulScheduler and
+// HopperMultipleMatmulScheduler
+class MultipleMatmulScheduler {
```

Comment:
It could be in another PR but we could probably remove the `Multiple` from the names of these classes. It was needed while we had two matmul schedulers temporarily but not any more.